### PR TITLE
Update Terra ml4h featured workspace story

### PR DESCRIPTION
**Pull request by deflaux**
Bring this repo up to date with the version of the notebooks in Terra: https://app.terra.bio/#workspaces/uk-biobank-sek/ml4cvd-featured-workspace/notebooks

Still TODO, add generate_synthetic_tabular_data.ipynb to the repo. But if I recall correctly, you folks plan to let me know about a few more desired changes you would like to see in the synthetic data.

### CODE REVIEW

File: notebooks/terra_featured_workspace/review_model_results_interactive.ipynb [hunk 1]
```diff
@@ -4,11 +4,15 @@
...
-    "# Review model results - Step 1 - Identify a sample to review\n",
+    "# Review model results\n",
     "\n",
-    "In this notebook, we review phenotypes and model results to identify one or more sample ids of interest.\n",
+    "In this notebook, we review phenotypes and model results to identify where our model has room for improvement.\n",
     "\n",
-    "In a separate notebook, we'll take a closer look at the input data to the model for the identified sample(s)."
+    "<div class=\"alert alert-block alert-info\">\n",
+    "\n",
+    "<p>Machine learning engineers work <b>iteratively</b> with domain experts, such as <b>cardiologists</b>, to understand how well the model is performing, and to enumerate the situtations in which it needs <b>improvement</b>.</p>\n",
```
Comment (paolodi) [inline notebooks/terra_featured_workspace/review_model_results_interactive.ipynb:13]: Little typo: `situtations`

File: notebooks/terra_featured_workspace/review_model_results_interactive.ipynb [hunk 2]
```diff
@@ -93,15 +84,78 @@
...
+    "\n",
+    "<b>Observations that could be made:</b>\n",
+    "    <ol>\n",
+    "        <li>We can see that the model often gets the P axis prediction wrong for higher BMI individuals. Plot <kbd>p_axis_delta</kbd> vs. <kbd>bmi</kbd> using Facets Dive and you will see the skew.</li>\n",
+    "        <li>We can see that the model often gets the QT interval prediction wrong when the person did not exercise very hard. There is more error for lower speed bike ECGs because they typically result in a lower average load during the exercise phase. Plot <kbd>qt_interval_prediction</kbd> vs. <kbd>qt_interval_actual</kbd> coloring by <kbd>ecg_exercise_program</kbd> using Facets Dive and you will see the skew.</li>\n",
+    "        <li>We can see that the model often gets the liver visceral adipose fraction (proton density fat) prediction wrong for higher BMI individuals. Plot <kbd>proton_density_fat_delta</kbd> vs. <kbd>bmi</kbd> using Facets Dive and you will see the skew.</li>\n",
+    "    </ol>\n",
+    "\n",
+    "<p><b>Outcome:</b> after review of model inputs and predictions, we realize that BMI and ecg exercise program should be added to the model inputs.</p>\n",
```
Comment (paolodi) [inline notebooks/terra_featured_workspace/review_model_results_interactive.ipynb:137]: Nit: capitalize `ecg`

File: notebooks/terra_featured_workspace/review_one_sample_interactive.ipynb [hunk 3]
```diff
@@ -4,9 +4,15 @@
...
    "source": [
-    "# Review model results - Step 2 - Review one sample\n",
+    "# Review one sample\n",
     "\n",
-    "In this notebook, we take a closer look at the model inputs and outputs for a single sample."
+    "In this notebook, we take a closer look at the model inputs and outputs for a single sample.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "\n",
+    "<p>Machine learning engineers work <b>iteratively</b> with domain experts, such as <b>cardiologists</b>, to understand how well the model is performing, and to enumerate the situtations in which it needs <b>improvement</b>.</p>\n",
```
Comment (paolodi) [inline notebooks/terra_featured_workspace/review_one_sample_interactive.ipynb:13]: Little typo: `situtations`

File: notebooks/terra_featured_workspace/image_annotations_demo.ipynb [hunk 4]
```diff
@@ -6,7 +6,15 @@
    "source": [
     "# Image annotations for a batch of samples\n",
     "\n",
-    "Using this notebook, cardiologists are able to quickly view and annotate MRI images for a batch of samples. These annotated images become the training data for the next round of modeling."
+    "In this notebook, for a batch of samples we view and annotate MRI images by drawing polygons on the images.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "\n",
+    "<p>Machine learning engineers work <b>iteratively</b> with domain experts, such as <b>cardiologists</b>, to understand how well the model is performing, and to enumerate the situtations in which it needs <b>improvement</b>.  In some cases cardiologists provide direct input to the model in the form of annotated images:\n",
```
Comment (paolodi) [inline notebooks/terra_featured_workspace/image_annotations_demo.ipynb:13]: Little typo: `situtations`

**Review by paolodi — APPROVED** [review]
@deflaux I love the new text (I added a couple of comments inline where I could spot a typo), and these notebooks are so informative! Please go ahead and merge.

P.S.: @gsarma and I will get back to you regarding other potential model confounders for the "Review model results" notebook

**Comment by deflaux** [comment]
Thanks Paolo! I tried to fix the typos in my private fork but those commits do not seem to be reflected here.

I will fix those in a subsequent PR. Thank you!!

**Pull request merged**